app/src/userpages/components/CanvasPage/List/index.jsx

```diff
@@ -282,11 +281,6 @@ class CanvasList extends Component<Props, State> {
                                     link={`${links.editor.canvasEditor}/${canvas.id}`}
                                     dropdownActions={this.getActions(canvas)}
                                     image={<CanvasPreview className={styles.PreviewImage} canvas={canvas} />}
-                                    onMenuToggle={(open) => {
-                                        if (open) {
-                                            this.loadCanvasPermissions(canvas.id)
```

Comment:
The `loadCanvasPermissions` and `hasWritePermission` method can be removed if not used... and the related props (`getCanvasPermissions`, `fetchingPermissions`, etc) can be removed also.